modules/troubleshooting-network-observability-gprc-message-larger-than-max.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * networking/network_observability/troubleshooting-network-observability.adoc
+
+:_content-type: PROCEDURE
+[id="nw-lokistack-rate-limits_{context}"]
+= gRPC received a message larger than the maximum size
+
+The Loki distributor receives messages in bulk. These messages are exchanged between components that have a maximum size in bytes. gRPC may receive a message larger than its maximum size. You can increase the size of `grpc_server_max_recv_msg_size` and `grpc_server_max_send_msg_size`.
```

Comment:
gRPC is a framework. The distributor is the failing component in that case.

```suggestion
The Loki distributor receives messages in bulk. These messages are exchanged between components that have a maximum size in bytes. Loki distributor may receive a message larger than its maximum configured size. You can increase the size of `grpc_server_max_recv_msg_size` and `grpc_server_max_send_msg_size`.
```

Reply:
Addressed